addons/addon-base-raas/packages/base-raas-services/lib/schema/update-aws-accounts.json

```diff
@@ -26,6 +26,13 @@
       "type": "string",
       "pattern": "^[0-9]{12}$"
     },
+    "cfnStackName": {
+      "type": "string",
+      "maxLength": 255
+    },
+    "mainRegion:": {
```

Comment:
Do we allow the member account region to be different from main account? Not sure if region customization is supported right now...

Reply:
thats a good question that I hadn't thought of, I can find out